Verify that the RAT Type is WLAN or EUTRAN

vikg-fb opened the pull request:
Summary: Per 3GPP TS 29.272 section 5.2.1.1.3 (ULRs) and 3GPP TS 29.273 section 8.1.2.1.2 (MARs) the RAT Type AVP needs to be checked to ensure that it is allowed. Currently, we will allow it to be WLAN and EUTRAN.

Differential Revision: D14227395

anooptomar29 commented on feg/gateway/services/swx_proxy/servicers/swx_definitions.go:
```diff
@@ -18,7 +18,8 @@ const (
 	// 3GPP 29.273 5.2.3.6
-	RadioAccessTechnologyType_WLAN = 0
+	RadioAccessTechnologyType_WLAN   = 0
+	RadioAccessTechnologyType_EUTRAN = 1004
```
Lets keep only non-3GPP RAT types for SWx . So for now lets keep WLAN only.

anooptomar29 commented on feg/gateway/services/s6a_proxy/servicers/s6a_definitions.go:
```diff
@@ -14,6 +14,12 @@ import (
 
+const (
+	// 3GPP 29.273 5.2.3.6
+	RadioAccessTechnologyType_WLAN   = 0
```
For S6a lets keep only 3GPP RAT types . So for now let's keep E-UTRAN only.

vikg-fb committed:
Remove unused import in eap_provider.proto

Differential Revision: D14251256

fbshipit-source-id: db6c9772f982f658309e6a7137547990eb36af48

vikg-fb committed:
Verify that the RAT Type is WLAN or EUTRAN (#11)

Summary:
Pull Request resolved: https://github.com/facebookincubator/magma/pull/11

Per 3GPP TS 29.272 section 5.2.1.1.3 (ULRs) and 3GPP TS 29.273 section 8.1.2.1.2 (MARs) the RAT Type AVP needs to be checked to ensure that it is allowed. Currently, we will allow it to be WLAN and EUTRAN.

Reviewed By: mpgermano

Differential Revision: D14227395

fbshipit-source-id: 7f524e359b52edbf8a860de25995e97029d85060
